contentcuration/contentcuration/views/base.py

```diff
@@ -204,16 +211,37 @@ def get_user_edit_channels(request):
     edit_channels = request.user.editable_channels.exclude(deleted=True)\
         .select_related('main_tree').prefetch_related('editors')\
         .defer('trash_tree', 'clipboard_tree', 'staging_tree', 'chef_tree', 'previous_tree', 'viewers')
+    edit_channels = _apply_channel_filters(edit_channels, request.query_params)
     channel_serializer = AltChannelListSerializer(edit_channels, many=True)
     return Response(channel_serializer.data)
 
 
+@api_view(['GET'])
+@authentication_classes((SessionAuthentication, BasicAuthentication, TokenAuthentication))
+@permission_classes((IsAuthenticated,))
+def get_user_channel_sets(request):
+    sets = request.user.channel_sets.prefetch_related('secret_token__channels', 'editors').select_related('secret_token')
```

Comment:
Do you think it's also good to prefetch the channels associated with this channel set too? Is that possible, or does `(select|prefetch)_related` not support that?